src/coreclr/scripts/genEventPipe.py

```diff
@@ -72,26 +72,6 @@ def generateMethodSignatureWrite(eventName, template, extern, runtimeFlavor):
     sig_pieces.append(")")
     return ''.join(sig_pieces)
 
-def includeProvider(providerName, runtimeFlavor):
```

Comment:
Isn't this method still called in this source, generateClrEventPipeWriteEventsImpl?